pages/organization.tsx

```diff
@@ -1,5 +1,47 @@
-import React from "react";
+import DirectoryOrgHeader from '../components/DirectoryOrgHeader';
```

Comment:
I just noticed this - is it expected that the DirectoryOrgHeader name is used for this component in the org page?

Reply:
Just renamed that to just "OrgHeader"